src/components/scenes/GettingStartedScene.tsx

```diff
@@ -83,8 +82,8 @@ export const GettingStartedScene = (props: Props) => {
   const localUsers = useWatch(context, 'localUsers')
   const hasLocalUsers = localUsers.length > 0
 
-  const [isFinalSwipeEnabled, setIsFinalSwipeEnabled] = React.useState<Boolean>(asFbRemoteConfig({}).swipeLastUsp)
-  const [createAccountType, setCreateAccountType] = React.useState<CreateAccountType>(asFbRemoteConfig({}).createAccountType as CreateAccountType)
+  const [isFinalSwipeEnabled, setIsFinalSwipeEnabled] = React.useState<Boolean>(true)
```

Comment:
Remove type arguments and lean on inference from the default value. Also `Boolean` is not the same as `boolean` type.